test/helper/attribute.ex

```diff
@@ -4,4 +4,8 @@ defmodule Jerboa.Test.Helper.Attribute do
   def total(x) do
     x |> Keyword.values |> Enum.sum
   end
+
+  def value_length(<<_::16, byte_length::16, _::size(byte_length)-bytes>>), do: 8 * byte_length
```

Comment:
Actually the function, despite being beautiful in its pattern matching, does a little bit more than it claim to do:
1) it check if the size of the payload is correct
2) returns that size.

But the name only declares it is responsible for 1). Thoughts?